chrome/chrome-tests.ts

```diff
@@ -26,6 +26,7 @@ function bookmarksExample() {
         return list;
     }
     function dumpNode(bookmarkNode, query) {
+        var span = $('<span>');
```

Comment:
It's a snippet from one of the Chrome example extensions, but it's also broken because span was used outside of the block in which it was declared, so strict null checking rightfully flags that as an error. I just moved the declaration out one block.